### DISCUSSION TITLE
透過深度神經網絡 ExoMiner 發現新發現的301 顆新系外行星

### DISCUSSION THREAD

**Discussion by EricAgumonJou** [1101-GS4719-C/Python-Community-GS4719-C#168] [General]
深度神經網絡是機器學習方法，可在提供data充足的情況下進行自動學習等任務。 
ExoMiner 是一種新設計的深度神經網絡，它利用了 NASA 的超級電腦（Pleiades），可以將真正的系外行星與不同類型的冒名星（imposters）或“誤報（false positives）”區分開來。

它的設計靈感來自人類專家用來確認新系外行星的各種測試和特性。
ExoMiner會通過過去確認的系外行星和誤報（false positives）的案例中學習。

截至 NOVEMBER 23, 2021 目前已透過ExoMiner成功尋找到301顆新系外的行星。
![Exoplanet-Artist-Concept-for-ExoMiner-2048x1152](https://user-images.githubusercontent.com/91121500/143186607-416d60ac-4ae6-497c-bf84-e40f12b1a0e5.jpg)

資料來源：https://scitechdaily.com/a-whopping-301-newly-confirmed-exoplanets-discovered-with-new-deep-neural-network-exominer/

**Comment by EricAgumonJou** [comment]
ExoMiner 能有效幫助擅長處理數據和破解行星奧秘的科學家。該數據則由 NASA 的Kepler spacecraft and K2航天器及其後續收集而來的的。 

對於Kepler航天器，在其視野中能捕捉到數千顆恆星，而每顆恆星的周圍都有可能發現多顆潛在的系外行星，然而研究大量數據集是一項非常耗時的任務，故使用ExoMiner能快速解決此一難題。

**Comment by EricAgumonJou** [comment]
補充一下如何尋找新的新系外行星：

當一顆行星直接穿過我們和該行星所屬的恆星間時，我們會看到該顆恆星略微變暗，因為這顆行星擋住了一部分光。 
測量星光中的傾角可用來識別系外行星（transit method）。

![Transit-Method](https://user-images.githubusercontent.com/91121500/143188070-75f71742-7dc2-4ae5-a156-7da7d8e84693.gif)

科學家以此資料繪製“光度曲線”圖（如上圖），該圖為恆星亮度隨時間變化作圖。
科學家可從中看到行星阻擋了恆星光的百分比以及行星穿過恆星盤所需的時間，這些資料可幫助他們估計行星與恆星的距離及其質量。
若透過ExoMiner則可更快速地分析及處理資料。

**Comment by Jie2222** [comment]
最近多次聽到機器學習運用在太空領域的案例，酷

**Comment by wax75397** [comment]
我想這個方法說不定也能用來辨別小行星，透過直接與軌道數據庫比對，發現新的小行星

**Comment by xCk27x** [comment]
@EricAgumonJou 所以ExoMiner的工作是進行系外行星的的勘誤嗎 ? 就是把過往找到的系外行星的資料給他，讓他找哪些其實不是系外行星。還是說給他一些恆星的光度變化等資料，他就會自動幫你找出其中多少恆星。

**Comment by 107602505** [comment]
沒想到機器學習也能運用在太空上真的佩服，上面有提到ExoMiner 是一種新設計的深度神經網絡，可以將真正的系外行星與不同類型的冒名星（imposters）或“誤報（false positives）”區分開來，蠻好奇它的準確性有多高?

**Comment by Evan0979** [comment]
好認真還有附解說讚讚

**Comment by CPCpyt** [comment]
我覺得好棒，真的是非常有內涵的文章，
感覺又被知識灌溉了不少，實在令人佩服。

**Comment by ruthchen1020** [comment]
我覺得很厲害，PO感覺對這方面頗有研究，用機器學習來做比對真的可以少很多工作量。
